src/components/Bullet.tsx

```diff
@@ -206,6 +447,9 @@ const Bullet = ({
     return styles?.color
   })
 
+  const isExpanded = useSelector(state => !!state.expanded[hashPath(path)])
+  const isCursor = isCursorParent || isCursorGrandparent || isEditing || isExpanded
```

Comment:
Can you help me understand this `isCursor` variable? It appears to be true when the thought itself (`isEditing`), parent (`isCursorParent`), or grandparent (`isCursorGrandparent`) is the cursor, _or_ when the thought is expanded. I don't understand what those states have in common. Clearly a Bullet can be expanded without having the cursor on it.

Reply:
I wasn't really sure what to name it, isExpanded might make more sense, but that is used already.

it comes from these styles
```css
.child.expanded:not(.leaf) > .thought-container > .bullet .glyph .glyph-fg.triangle,
.child.editing:not(.leaf) > .thought-container > .bullet .glyph .glyph-fg.triangle,
.child.cursor-parent:not(.leaf) > .thought-container > .bullet .glyph .glyph-fg.triangle,
.child.cursor-grandparent:not(.leaf) > .thought-container > .bullet .glyph .glyph-fg.triangle 
```

Reply:
I see! Yes, that's literally if the bullet is expanded. We can call it `isBulletExpanded` for now.

I'm not sure why it's not covered by the first selector `.child.expanded:not(.leaf) > .thought-container > .bullet .glyph .glyph-fg.triangle` since `.editing` should always be `expanded`. But simplifying that is a different task. We'll focus on converting most the styles as-is before we do any serious refactoring or simplifying.

Reply:
maybe just `expand: true` would work